SonarQube.Bootstrapper/Resources.resx

```diff
@@ -159,9 +162,6 @@
   <data name="PhaseLabel_PreProcessing" xml:space="preserve">
     <value>Pre-processing</value>
   </data>
-  <data name="WARN_CmdLine_UrlRequired" xml:space="preserve">
-    <value>The SonarQube URL was not set so the default value {0} is going to be used. The URL can be specified in a settings file or on the command line (e.g. using /d:sonar.host.url=http://myserver:9000). </value>
-  </data>
```

Comment:
Now an error rather than a warning (although this will only occur if the default properties file has been deleted or edited to remove the host url property).